common/src/Assets/Texture.h

```diff
@@ -46,7 +46,9 @@ namespace TrenchBroom {
 
             size_t m_usageCount;
             bool m_overridden;
-            
+
+            GLenum m_format = GL_RGB;
```

Comment:
Please use the constructors to initialize members. I prefer to have initialization of members in one place.